lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -18,8 +18,8 @@ def install!
 
             UI.message "- Installing target `#{target.name}` #{target.platform}" do
               add_target
-              add_test_targets if target.contains_test_specifications?
               create_support_files_dir
+              add_test_targets if target.contains_test_specifications?
```

Comment:
This needs to be after `create_support_files_dir` because now we will be creating `.sh` files for test targets.